recap/types.py

```diff
@@ -139,14 +139,16 @@ def __eq__(self, other):
         )
 
     def validate(self) -> None:
-        if self.bytes_ < 1 or self.bytes_ > 9_223_372_036_854_775_807:
-            raise ValueError("bytes must be between 1 and 9,223,372,036,854,775,807")
+        if not self.variable and self.bytes_ is None:
+            raise ValueError("Fixed length bytes must have a length set")
+        if self.bytes_ is not None and self.bytes_ < 1:
+            raise ValueError("Bytes must be greater than or equal to 1")
```

Comment:
Why are you dropping the max length constraint here and for Bytes below?

Reply:
Mmm, good question. I'm not 100% sure what I was thinking here. The spec still says `64-bit signed integer | null`, so if bytes_ is set it should be enforced as such. Good catch.

Reply:
Ok, I pushed an updated PR here (force push squash). LMK if this looks OK.

- Went back to LONG_MAX for string/bytes/list length
- Moved list length min from 0 to 1 (in accordance with spec)
- Updated 0.3.0.json to reflect maximum=int_max 
- Left 0.3.0.md as is since it already reflected these rules